### PR TITLE
ChildProcess. Add props

### PR DESCRIPTION
**Pull request by wirwolf**
Add  spawnargs spawnfile and connected to ChildProcess
screenshot_20181224_130632

### CODE REVIEW

File: types/node/index.d.ts [hunk 1]
```diff
@@ -2732,6 +2732,9 @@ declare module "child_process" {
         stdout: stream.Readable;
         stderr: stream.Readable;
         stdio: [stream.Writable, stream.Readable, stream.Readable];
+        spawnargs: [string];
```
Comment (KSXGitHub) [inline types/node/index.d.ts]: Why `[string]` and not `string[]`? (This is not Haskell BTW)

File: types/node/index.d.ts [hunk 2]
```diff
@@ -2732,6 +2732,9 @@ declare module "child_process" {
...
+        spawnfile: string;
+        connected: string;
```
Comment (KSXGitHub) [inline types/node/index.d.ts]: Should `connected` be `boolean`?

**Comment by ThomasdenH** [comment]
Generally the types should only contain public API, so we should follow the documentation (unless it's outdated). [`connected`](https://nodejs.org/api/child_process.html#child_process_class_childprocess) is in the documentation and missing in the types, so it should be added. It is however a `boolean`. I don't think the other properties should be added to the types.

**Review by ThomasdenH — CHANGES_REQUESTED** [review]
A PR with just `connected: boolean;` would be fine.

File: types/node/index.d.ts (hunk 2 above)
Comment (Flarna) [reply types/node/index.d.ts]: connected is already present as boolean at line 2744

File: types/node/index.d.ts [hunk 3]
```diff
@@ -2732,6 +2732,8 @@ declare module "child_process" {
         stdout: stream.Readable;
         stderr: stream.Readable;
         stdio: [stream.Writable, stream.Readable, stream.Readable];
+        spawnargs: [];
```
Comment (KSXGitHub) [inline types/node/index.d.ts]: I'm not sure if you are still expecting this to be merged, but why `[]`?

**Review by SimonSchick — CHANGES_REQUESTED** [review]
These are not in the official API docs: https://nodejs.org/api/child_process.html

**Pull request closed without merging**